ParselyTrackerTests/SamplerTests.swift

```diff
@@ -49,21 +49,40 @@ class SamplerTests: ParselyTestCase {
 
     func testBackoff() {
         let initialInterval = samplerUnderTest!.heartbeatInterval
-        let expectedUpdatedInterval: TimeInterval = TimeInterval(13.65)
-        let assertionTimeout:TimeInterval = initialInterval + TimeInterval(2)
+        // Ensure value matches magic number from production code
+        XCTAssertEqual(initialInterval, TimeInterval(10.5))
         
+        // Track an even, then make the test runner wait for the heartbeat interval plus some extra
```

Comment:
typo, `Track an even(t)`

Reply:
Thanks for catching this @cwisecarver! 🙇‍♂️ In other projects, we use GitHub's `suggestion` block a lot. Suggestions are always welcome 😄  

```suggestion
        // Track an event, then make the test runner wait for the heartbeat interval plus some extra
```